scripts/.trivyignore

```diff
@@ -1,2 +1,5 @@
 # Waiting for upstream prometheus/ promtool to patch the vulnerability for "go-restful"
 CVE-2022-1996
+# Ruby (gemspec) vulnerabilities for logstash container
+# CVE-2022-25648 # git
```

Comment:
I think these two lines need to be uncommented

Reply:
ah good catch!